app/src/main/java/com/github/sdp/mediato/model/User.java

```diff
@@ -0,0 +1,127 @@
+package com.github.sdp.mediato.model;
+
+import com.github.sdp.mediato.errorCheck.Preconditions;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class User {
+    public static final int LIMIT_LOCATION_SIZE = 2;
+    public static final String FAVOURITES_COLLECTION = "Favourites";

```

Comment:
I just committed changes where I suggest a similar implementation in [8b153b2]. I created the Collection class and CollectionType enum but left the Map<String, Collection> associated to the user since I think it can be useful for searching a collection by its name.